### DISCUSSION TITLE
Upgrading OpenVAS ERROR - gvm-libs 11.0.1

### DISCUSSION THREAD

**Discussion by devMarc5598** [Secure-Compliance-Solutions-LLC/GVM-Docker#210] [Q&A]
Hi,
which steps are necessary to upgrade the openvas and gvm-libs to the latest version?
Just pull and run the latest image does not work...

I currently using the OpenVAS gvm-libs 11.0.1 and i get the following Vulnerability in my latest reports:
“Report outdated / end-of-life Scan Engine / Environment (local)”
It says that my currently installed GVM Libraries (gvm-libs) version is 11.0.1 and that I should update the gvm-libs to the latest available GVM Libraries version 20.8.0.

My GVM versions:
gsad: Greenbone Security Assistant 9.0.1
gvmd: Greenbone Vulnerability Manager 9.0.1
openvas-scanner: OpenVAS 7.0.1
gvm-libs: gvm-libs 11.0.1

**Comment by Dexus** [comment]
pull the latest image and recreate a new container?
